lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -0,0 +1,288 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone', 'js/vendor/backbone-super',
+    ], function (gettext, $, _, Backbone) {
+
+        var messageRevertDelay = 4000;
+        var AccountSettingsFieldViews = {};
+
+        AccountSettingsFieldViews.FieldView = Backbone.View.extend({
+
+            className: function () {
+                return "account-settings-field " + "account-settings-field-" + this.options.valueAttribute;
+            },
+
+            tagName: 'div',
+
+            errorMessage: '<i class="fa fa-exclamation-triangle message-error"></i>' + gettext('An error occurred, please try again.'),
```

Comment:
add an ```aria-hidden=true``` attribute on the icon font and a screen reader accessible label i.e. ```<span class="sr">Error:</span>``` as  a sibling to the icon font (between the icon and the actual error msg)